cmake warning

I get this warning when I run the command `make` on nobara (fedora fork):
```
CMake Deprecation Warning at cmake-build-release/_deps/yaml-cpp-src/CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
```
cmake version: 27.7.2

With CMake 4.0 this is no longer a warning, but an error

```
CMake Error at cmake-build-release/_deps/yaml-cpp-src/CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
```

This can be fixed by increasing CMake minimum version

https://github.com/HarbourMasters/Torch/blob/8e56ea0294973b7c9f8f077a131b40fce13b8489/CMakeLists.txt#L1

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.30)
 project(torch)
 include(FetchContent)
 
@@ -43,7 +43,7 @@ if(USE_STANDALONE)
     FetchContent_GetProperties(libgfxd)
 
     if(NOT libgfxd_POPULATED)
-        FetchContent_Populate(libgfxd)
+        FetchContent_MakeAvailable(libgfxd)
         include_directories(${libgfxd_SOURCE_DIR})
         set(LGFXD_SRC gfxd.c uc_f3d.c uc_f3db.c uc_f3dex.c uc_f3dexb.c uc_f3dex2.c)
         foreach (LGFXD_FILE ${LGFXD_SRC})

